components/tools/OmeroPy/src/omero/plugins/metadata.py

```diff
@@ -395,6 +396,20 @@ def get_anns(md):
             indent = 0
         self._output_ann(md, get_anns, args.parents, indent)
 
+    def testtables(self, args):
+        "Tests whether tables can be created and initialized"
+        client = self.ctx.conn(args)
+
+        sf = client.getSession()
+        sr = sf.sharedResources()
+        table = sr.newTable(1, 'testtables')
```

Comment:
table is never closed

Reply:
Agreed. `TablePrx` instances should be closed in try/finally blocks and in this case a further finally should delete the file that was created.